app/controllers/support/ConfigChecker.scala

```diff
@@ -0,0 +1,98 @@
+package controllers.support
+
+import java.util.UUID
+
+import com.m3.octoparts.aggregator.handler.HttpPartRequestHandler
+import com.m3.octoparts.model.config.{ HttpPartConfig, ParamType, PartParam }
+import com.netaporter.uri.Parameters.Param
+import com.netaporter.uri.dsl._
+import play.api.{ Mode, Play }
+import play.api.i18n.{ Lang, Messages }
+
+trait ConfigChecker[T] {
```

Comment:
For now the only implementations have T = HttpPartConfig, but in the near future I am sure there will be more